-- Add the zbus dependency for FreeBSD.
-- zbus was only declared for Linux, but FreeBSD uses the same DBus backend.
--- cargo-crates/tauri-plugin-single-instance-2.3.6/Cargo.toml.orig	1970-01-01 00:00:01 UTC
+++ cargo-crates/tauri-plugin-single-instance-2.3.6/Cargo.toml
@@ -80,7 +80,7 @@ version = "0.1"
 [dependencies.tracing]
 version = "0.1"
 
-[target.'cfg(target_os = "linux")'.dependencies.zbus]
+[target.'cfg(any(target_os = "linux", target_os = "freebsd"))'.dependencies.zbus]
 version = "5.9"
 
 [target.'cfg(target_os = "windows")'.dependencies.windows-sys]
